.github/workflows/dependancy-review.yml

```
@@ -0,0 +1,20 @@
+
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Repository'
+        uses: actions/checkout@v4
+
+      - name: 'Dependency Review'
+        uses: actions/dependency-review-action@v4
+        with:
+          severity-threshold: high
```

actionlint (1.7.12)

[error] input "severity-threshold" is not defined in action "actions/dependency-review-action@v4". available inputs are "allow-dependencies-licenses", "allow-ghsas", "allow-licenses", "base-ref", "comment-summary-in-pr", "config-file", "deny-groups", "deny-licenses", "deny-packages", "external-repo-token", "fail-on-scopes", "fail-on-severity", "head-ref", "license-check", "repo-token", "retry-on-snapshot-warnings", "retry-on-snapshot-warnings-timeout", "show-openssf-scorecard", "show-patched-versions", "vulnerability-check", "warn-on-openssf-scorecard-level", "warn-only"

(action)